db/migrate/20150818184658_create_tenant_quota.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTenantQuota < ActiveRecord::Migration
+  def change
+    create_table :tenant_quotas do |t|
+      t.belongs_to :tenant, type: :bigint
+
+      t.string     :name
+      t.string     :unit
+      t.float      :value
+
```

Comment:
@mkanoor We were thinking that we can make an entry in the en.yml keyed by the value in name for the description because we will be the ones defining the supported quotas.